jake/_version.py

```diff
@@ -0,0 +1 @@
+__version__="0.0.1"
```

Comment:
Neato!

May have to keep this in mind if we ever get fancy with CI, automated releasing, etc.

Reply:
I imagine this will change significantly as we do something like that, but for now, it works!